site/src/api/typesGenerated.ts

```diff
@@ -29,6 +29,13 @@ export interface AgentGitSSHKey {
   readonly private_key: string
 }
 
+// From codersdk/metrics.go
+export interface AgentStatsReportResponse {
+  readonly num_comms: number
+  readonly rx_bytes: number
+  readonly tx_bytes: number
```

Comment:
What do `rx_bytes` and `tx_bytes` stand for? Can we add a comment somewhere? Or explain in the PR description?